runelite-client/src/main/java/net/runelite/client/plugins/screenmarkers/ScreenMarkerPlugin.java

```diff
@@ -174,18 +232,141 @@ public void startCreation(Point location)
 			true
 		);
 
+		creatingScreenMarker = true;
+
 		// Set overlay creator bounds to current position and default size
 		startLocation = location;
 		overlay.setPreferredLocation(location);
 		overlay.setPreferredSize(DEFAULT_SIZE);
-		creatingScreenMarker = true;
+
```

Comment:
Delete empty new line

Reply:
Why was this moved in the first place? Seems unnecessary.